.travis.yml

```diff
@@ -57,7 +57,7 @@ install:
   - "inveniomanage config create secret-key"
   - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
   - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
+  - "inveniomanage config set PACKAGES ['invenio_tags', 'invenio_records', 'invenio_accounts', 'invenio_base']"
```

Comment:
```
2.34s$ inveniomanage config set PACKAGES ['invenio_tags', 'invenio_records', 'invenio_accounts', 'invenio_base']
usage: inveniomanage [-?]
                     {shell,knowledge,database,clean,check-for-software-updates,runserver,records,version,bower,upgrader,collect,apache,detect-system-details,formatter,config,show-urls,assets}
                     ...
inveniomanage: error: too many arguments
```

you need to better escape the arguments or use `.kwalitee.invenio.cfg` trick from other repositories.